portfolio.js

```diff
@@ -0,0 +1,25 @@
+const hambergur = document.querySelector('.allign');
+const contacts = document.querySelector('.contacts');
+const closes = document.querySelector('.close');
+const main = document.querySelector('.main');
+hambergur.addEventListener('click', () => {
+  hambergur.classList.toggle('close');
+  closes.style.display = 'block';
+  hambergur.style.display = 'none';
+  contacts.style.display = 'block';
+  main.style.opacity = '0';
+});
+
+closes.addEventListener('click', () => {
+  closes.classList.toggle('allign');
+  hambergur.style.display = 'block';
+  closes.style.display = 'none';
+  contacts.style.display = 'none';
+  main.style.opacity = '1';
+});
+contacts.addEventListener('click', () => {
```

Comment:
- [ ] [Optional] When the user clicks any of the mobile menu options, a correct part of the page should be displayed and the menu should disappear, kindly implement these functionalities it's a requirement